src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -313,19 +290,6 @@ export class ItemByIdComponent implements OnDestroy, BeforeUnloadComponent, Pend
       this.initialAnswerDataSource.setInfo(route, isTask);
     }),
 
-    merge(
-      this.itemDataSource.state$.pipe(readyData()),
-      this.fullFrameContent$.pipe(distinctUntilChanged()),
-      this.taskTabs$,
-      fromEvent(window, 'resize'),
-    ).pipe(
-      delay(0),
-      map(() => this.contentContainer?.nativeElement.offsetTop || 0),
-      distinctUntilChanged(),
-    ).subscribe(contentContainerTop =>
-      this.contentContainerTop = contentContainerTop
-    ),
```

Comment:
wasn't it there to recompute the task size when tab change or window time changed? how is it related with the forum? Are you sure you are not breaking anything with that?

There should be test scenarios showing the behavior is still the same.

Reply:
It was done only for to set correct height of the thread when content is changed like task size, windows size etc https://github.com/France-ioi/AlgoreaFrontend/pull/1447/files#diff-bd35281dfe1d9124ef85c9a04f336b5d751ea8b33f624a97252f2afc236f83e2L221

I've add test case